ipyparallel/apps/ipclusterapp.py

```diff
@@ -572,35 +572,42 @@ def start():
 
 class IPClusterNBExtension(BaseIPythonApplication):
     """Enable/disable ipcluster tab extension in Jupyter notebook"""
-    
+
     name = 'ipcluster-nbextension'
-    
+
     description = """Enable/disable IPython clusters tab in Jupyter notebook
-    
+
     for Jupyter Notebook >= 4.2, you can use the new nbextension API:
-    
+
     jupyter serverextension enable --py ipyparallel
     jupyter nbextension install --py ipyparallel
     jupyter nbextension enable --py ipyparallel
     """
-    
+
     examples = """
     ipcluster nbextension enable
     ipcluster nbextension disable
     """
     version = __version__
-    
+    user = Bool(False, "Apply the operation only for the given user").tag(
+        config=True)
+    # flags = {}
+    flags = Dict(dict(user= (
+        {'IPClusterNBExtension': {'user': True}},
+        "Apply the operation only for the given user",
+    )))
```

Comment:
I'm confused as of why you mix dict literals with calling `dict`, is there a reason ?